lms/services/dashboard.py

```diff
@@ -219,8 +219,7 @@ def get_assignment_roster(
             # If rostering is enabled and we do have the data, use it
             query = self._roster_service.get_assignment_roster(
                 assignment,
-                role_scope=RoleScope.COURSE,
-                role_type=RoleType.LEARNER,
+                include_role=(RoleScope.COURSE, RoleType.LEARNER),
```

Comment:
Most of the changes is this, from two parameters to a tuple.

I might create a type alias when we add `exclude_role`.